modules/op-validating-pull-requests-using-GitHub-interceptors.adoc

```diff
@@ -0,0 +1,87 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-validating-pull-requests-using-GitHub-interceptors_{context}"]
+= Validating pull requests using GitHub Interceptors
+
+Using a GitHub Interceptor, you can validate processing of pull requests based on GitHub owners configured for a repository. The GitHub Interceptor processes a pull request only if the user name is listed as an owner or a comment is issued by an owner of the repository. This validation helps you to prevent unnecessary execution of a `PipelineRun` or `TaskRun` object.
+
+[NOTE]
+====
+Owners are configured in an `owners` file at the root of the repository.
+====
+
+.Prerequisites
+* You have installed the {pipelines-title} Operator.
+
+.Procedure
+. Create a secret string value.
+. Configure the GitHub webhook with that value.
+. Create a Kubernetes secret containing your secret value named `secretRef`.
+. Pass the Kubernetes secret as a reference to your GitHub Interceptor.
+. Create an `owners` file and add the list of approvers into the `approvers` section.
+. Perform one of the following steps: 
+* For a public GitHub repository, set the value of the `githubOwners` parameter to `true` in the YAML configuration file shown below:
++
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1beta1
+kind: EventListener
+metadata:
+  name: github-owners-listener
+spec:
+  triggers:
+    - name: github-listener
+      interceptors:
+        - ref:
+            name: "github"
+            kind: ClusterInterceptor
+            apiVersion: triggers.tekton.dev
+          params:
+            - name: "secretRef"
+              value:
+                secretName: github-secret
+                secretKey: secretToken
+            - name: "eventTypes"
+              value: ["pull_request", "issue_comment"]
+            - name: "githubOwners"
+              value: 
+                enabled: true
```

Comment:
No information about `checkType` 
cc: @savitaashture

Reply:
Added the details about `checkType`